optaplanner-website-assembly/src/main/assembly/assembly-website.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<assembly xmlns="http://maven.apache.org/ASSEMBLY/2.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://maven.apache.org/ASSEMBLY/2.0.0 http://maven.apache.org/xsd/assembly-2.0.0.xsd">
+
+  <id>assembly-optaplanner-website</id>
+  <formats>
+    <format>dir</format>
+  </formats>
+
+  <includeBaseDirectory>false</includeBaseDirectory>
+
+  <dependencySets>
+    <dependencySet>
+      <includes>
+        <include>org.optaplanner:optaplanner-website-docs:zip</include>
+      </includes>
+      <useProjectArtifact>false</useProjectArtifact>
+      <outputDirectory/>
+      <unpack>true</unpack>
+    </dependencySet>
+    <dependencySet>
+      <includes>
+        <include>org.optaplanner:optaplanner-website-main:zip</include>
+      </includes>
+      <useProjectArtifact>false</useProjectArtifact>
+      <outputDirectory/>
+      <unpack>true</unpack>
+    </dependencySet>
+  </dependencySets>
```

Comment:
Both website-main and website-docs's output is being zipped by assemblies there and then being unzipped by this assembly just to zip the aggregate.

Because website-main, website-docs and website-assembly are in the same repository (optaplanner-website), I'd argue it's probably fine to just use filesets and to relatively go get their output. Something like:
  https://github.com/kiegroup/optaplanner/blob/7.59.x/optaplanner-distribution/src/main/assembly/assembly-optaplanner.xml#L78
Even though that advises not to do for generated files, that advise is probably wrong (or stale), as KIE do it for years in release scripts IIRC without any issues.
Anyway, if we do that here, that saves 2 zippings and 2 unzipping, which should be measurable in build time.

Reply:
I know, on the other hand, navigating into other modules' directories directly is best to be avoided. I will take a look at how much time the zipping & unzipping actually take.